README.md

```diff
@@ -161,6 +161,26 @@ Examples: `Worked today 02:35, week 10:35` (hours:minutes).
 If timesheet line has empty time_to field,
 it's considered as current time.
 
+## JIRA completion
+
+This plugin helps you complete JIRA issue number by fuzzy matching issue title.
+To display completion popup, type "JI" and press `Alt-/`. Keep typing to further
+narrow candidates down. Press `Tab` to insert issue number:
+
+![JIRA completion](images/jira_completion.png)
+
+There is a caveat, though: request to JIRA server is currently synchronous, so
+it will block the editor. For this reason it is currently not activated
+automatically. You need to press `Alt-/` to request and display JIRA candidates
```

Comment:
Is `Alt+/` default key for suggestion? It's Ctrl+Space on my Mac. May be adjust docs?